tests/integration/cartography/intel/kandji/test_kandji.py

```diff
@@ -0,0 +1,27 @@
+import cartography.intel.kandji
+import tests.data.kandji.devices
+
+TEST_UPDATE_TAG = 123456789
+
+
+def test_load_kandji_devices_data(neo4j_session):
```

Comment:
You can copy this test: https://github.com/lyft/cartography/blob/7601060cca2bd73105dfde059659f2d985958ad1/tests/integration/cartography/intel/aws/test_emr.py#L25-L41

Using `check_nodes()` will help save some typing.


Also since _load_devices is the function being tested here, we shouldn't wrap it in another function. We normally use this _ensure_blah_blah() pattern in cases where we need the test graph to be pre-populated with other data but the actual function under test is something else.